gax/src/main/java/com/google/api/gax/rpc/Callables.java

```diff
@@ -215,4 +217,13 @@ private BatchingCreateResult(
     return new OperationCallableImpl<>(
         initialCallable, scheduler, longRunningClient, operationCallSettings);
   }
+
+  private static boolean areRetriesDisabled(
+      Collection<StatusCode.Code> retryableCodes, RetrySettings retrySettings) {
+    // RetryingCallable affects the rpc deadline of the initial attempt and the number of retries.
+    // If the rpc timeout is disabled and no further attempts can be made, then the retry
+    // infrastructure can be removed.
+    return retrySettings.getInitialRpcTimeout().isZero()
```

Comment:
`retrySettings.getInitialRpcTimeout().isZero()` in `&&` is suspicious. I have two questions:
1) What does `initialRpcTimeout == 0` means from sever side point of view (does it mean "no timeout", call takes as much time as it needs?)
2) Aren't `retrySettings.maxAttempts() == 1` and `retryableCodes.isEmtpy()` themselves are enough to disable retries (why is `initialRpcTimeout` relevant here)? I.e. why not just the following:
```java
return retrySettings.getMaxAttempts() == 1 || retryableCodes.isEmpty();
```
Otherwise, if `retrySettings.getInitialRpcTimeout().isZero() == false`, but  `retrySettings.getMaxAttempts() == 1 || retryableCodes.isEmpty() == true`, the whole expression gives `false`. Then, if something fails, the process goes into retrying infrastructure, but it immediately determines it as non-retriabel (one attempt is made, and one is already the maximum) and terminates.

Reply:
1. initialRpcTimeout doesn't have a meaning on the serverside. However, gax treats 0 as unset:
https://github.com/googleapis/gax-java/blob/master/gax/src/main/java/com/google/api/gax/rpc/AttemptCallable.java#L72-L75

2. Ok I'll remove it